bin/arno-iptables-firewall

```diff
@@ -47,15 +47,17 @@ fi
 
 # Check if the environment file exists and if so, load it
 #########################################################
-if [ -z "$ENV_FILE" ]; then
-  if [ -f /usr/local/share/arno-iptables-firewall/environment ]; then
+# Autodetect according to the path of the main script
+ENV_FILE="$(dirname $0)/../environment"
```

Comment:
So if the main script is `/usr/sbin/arno-iptables-firewall`, `ENV_FILE` is `/usr/environment` ?  Would this ever be true ?

Reply:
Ah good catch: Should be ENV_FILE="$(dirname $0)/../share/environment"